docs/organizations/verify-user-permissions.mdx

```diff
@@ -76,7 +76,7 @@ export function SettingsForm(){
 
 <Tabs type="framework" items={[ "RSC with has()", "RSC with auth().protect()", "RSC with <Protect>"]}>
   <Tab>
-    The following snippet uses `has()` to inspect a user's Permissions granularly. If the user doesn't have both `org:team_settings:manage` and `org:team_settings:read` Permissions, the component returns `null` instead of rendering its children.
+    The following snippet uses `has()` to inspect a user's Permissions granularly. If the user doesn't have the `org:team_settings:read` permission, the component returns `null` instead of rendering its children.
```

Comment:
```suggestion
    The following snippet uses `has()` to inspect a user's Permissions granularly. If the user doesn't have the `org:team_settings:read` Permission, the component returns `null` instead of rendering its children.
```